frontend/src/pages/ActivityReport/constants.js

```diff
@@ -42,15 +42,15 @@ export const recipientParticipants = [
 ];
 
 export const otherEntityParticipants = [
-  'Local/State Agency(ies)',
   'HSCO',
+  'Local/State Agency(ies)',
   'OCC Regional Office',
   'OHS Regional Office',
+  'Other',
```

Comment:
If this list is displayed in order, should "other" be last instead of alphabetical so the user would need to scroll past all the options first?

Reply:
Moved to bottom.